packages/data-context/test/unit/codegen/spec-options.spec.ts

```diff
@@ -242,6 +242,37 @@ describe('spec-options', () => {
         expect(result.codeGenType).to.eq('e2e')
         expect(result.fileName).to.eq(`TestName.foo.bar-copy-2.js`)
       })
+
+      context('file name contains special characters', async () => {
+        [
+          { condition: 'braces', fileName: '[...MyComponent].vue', expectedFileName: '[...MyComponent].cy.js', expectedComponentName: 'MyComponent' },
```

Comment:
Just to clarify, this PR only fixes the <name_of_component> part?

```js
import <name_of_component> from "'./[...blah].vue"
```

Looks good!

Reply:
Correct, just improving the conversion of the filename into a logical, legal JS var name. Expected filename conditions in this test are just there to ensure no regression was/is introduced